charts/prod_le.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: cert-manager.io/v1
```

Comment:
lets make a separate chart for cluster issuers. Here made a ticket for you in the board